integration-tests/enrich-kafka/config/iglu_resolver.json

```diff
@@ -9,7 +9,7 @@
         "vendorPrefixes": [ "com.snowplowanalytics" ],
         "connection": {
           "http": {
-            "uri": "http://iglucentral.com"
+            "uri": "http://iglucentral-dev.com.s3-website-us-east-1.amazonaws.com/feature/yauaa-1-0-4/"
```

Comment:
This line will be removed when [the iglu central PR](https://github.com/snowplow/enrich/pull/678) is merged.